DigitalLearningSolutions.Web/Views/Frameworks/Developer/CompetencyGroup.cshtml

```diff
@@ -27,27 +28,37 @@
     {
       <h1>
         Edit @Model.VocabSingular().ToLower() group
-      </h1>
+      </h1>
     }
     else
     {
       <h1>
         Add new @Model.VocabSingular().ToLower() group
-      </h1>
-
+      </h1>
     }
     @if (Model.DetailFramework.PublishStatusID == 3)
-    {
-      <partial name="Shared/_PublishedWarning" />
+    {
+      <partial name="Shared/_PublishedWarning" />
     }
-    <form method="post">
+    <form method="post">
+      @if (!ViewData.ModelState.IsValid)
+      {
+        <partial name="_ErrorSummary" />
+      }
       <nhs-form-group nhs-validation-for="CompetencyGroupBase.Name">
         <label class="nhsuk-label" id="competency-group-name-label" for="competency-group-name">
           @Model.VocabSingular() group name
         </label>
         <span nhs-validation-for="CompetencyGroupBase.Name"></span>
         <input class="nhsuk-input" asp-for="CompetencyGroupBase.Name" id="competency-group-name" name="Name" type="text" error-class-toggle="nhsuk-input--error" aria-describedby="competency-group-name-label">
       </nhs-form-group>
+      <nhs-form-group nhs-validation-for="CompetencyGroupBase.Description">
+        <label class="nhsuk-label" id="competency-description-label" for="competency-description">
```

Comment:
This should be for `... id="competency-group-description-label" for="competency-group-description"` (missing **group** in both places)